js/ui/pivot_grid/ui.pivot_grid.js

```diff
@@ -1265,6 +1265,10 @@ const PivotGrid = Widget.inherit({
         testElement.remove();
     },
 
+    _useFloatingWidthCorrection() { // T914454
```

Comment:
Maybe to rename to the _needFloatingWidthCorrection.

Reply:
Yep, fixed